.github/workflows/tests.yml

```diff
@@ -10,7 +10,7 @@ env:
 jobs:
   syntax:
     if: github.repository == 'Homebrew/brew'
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I think it's worth splitting this into another PR (as there's a failure below resulting from this).

Reply:
Yes, this will be reverted. I just changed it to test on CI (obviously it didn't quite work)